packages/plugin-pack/sources/packUtils.ts

```diff
@@ -239,11 +241,34 @@ export async function genPackList(workspace: Workspace) {
     }
   }
 
-  return await walk(workspace.cwd, {
+  const paths = await walk(workspace.cwd, {
     hasExplicitFileList,
     globalList,
     ignoreList,
   });
+
+  const files = paths.map(path => ({file: path, source: ppath.resolve(workspace.cwd, path)}));
+
+  if (workspace.cwd !== project.cwd) {
+    const workspaceHasLicense = paths.some(path => path.match(LICENSE_PATTERN));
```

Comment:
Hardcoding something about LICENSE files in this function feels a little too specific. Perhaps instead the files set via the `files` parameter should all be case insensitive?

Reply:
Hey @arcanis the regex is case-insensitive. Also, in this case, it's a little hard to reduce specificity because what we are doing is so specific :p. Would love any suggestion for alternatives.